elements/lisk-p2p/test/functional/network_fully_connected.ts

```diff
@@ -0,0 +1,653 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect } from 'chai';
+import { P2P, EVENT_CLOSE_OUTBOUND } from '../../src/index';
+import { wait } from '../utils/helpers';
+import { platform } from 'os';
+import { InboundPeer, OutboundPeer, ConnectionState } from '../../src/peer';
+
+describe('Fully connected network: The seedPeers list of each node contains the previously launched node', () => {
```

Comment:
`Fully connected` is misleading as well. I realise we test several general things. Shall we split it more? The file is clearly way too big.

Reply:
I think now we don't have to use `fully connected network` or `partially connected` because  its always a partial mesh now

Reply:
This was the existing description so I kept it as is. We can update the description to reflect the new network overlay.